package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Joseki",
+  "version": "1.0.0",
+  "description": "Easy way to organize events",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha test/server.spec.js",
+    "start": "gulp watch & nodemon server/server.js",
+    "postinstall": "./node_modules/bower/bin/bower install"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/VacantAardvark/joseki"
+  },
+  "author": [
+    "Derek Sakomoto",
+    "Eddie Kong",
+    "Kevin Huang",
+    "Steven Shyun"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/VacantAardvark/joseki/issues"
+  },
+  "homepage": "https://github.com/VacantAardvark/joseki",
+  "dependencies": {
+    "body-parser": "1.12.3",
+    "bower": "1.4.1",
+    "express": "4.12.3",
+    "mysql": "5.6.24",
+    "react": "^0.13.3",
```

Comment:
I would take out the ^ so that we don't run into any deployment issues in case there's a new version